packages/desktop-client/src/components/budget/index.js

```diff
@@ -331,7 +331,7 @@ class Budget extends PureComponent {
   };
 
   onShowActivity = (categoryName, categoryId, month) => {
-    this.props.history.push({
+    window.__history.push({
```

Comment:
Could this use `useHistory` or similar? It would be great to avoid the global unless absolutely necessary.